generator/.DevConfigs/aabf4e64-a7dd-4ee3-95a7-c8466c22a7e3.json

```diff
@@ -1,9 +1,10 @@
 {
   "core": {
     "changeLogMessages": [
-      "Improve user agent string construction in Marshaller.cs"
+      "Add new extension packages for CloudFront Signers and EC2 Decrypt Password to take a dependency on BouncyCastle.Cryptography"
     ],
-    "type": "patch",
+    "type": "minor",
```

Comment:
I think `type` here is ignored by our build system (since there's an `overrideVersion` as well). 

Also, is the `-preview` suffix required? For example, see the config we use for the first preview: https://github.com/aws/aws-sdk-net/blob/15a866102abf3763fa4c180c936a248d3b40c45b/generator/.DevConfigs/525da348-b9e8-4f4d-9dc7-47b2ee7f4716.json

Reply:
The preview actually failed the E2E test I ran last night, so I will remove it